index.bs

```diff
@@ -1218,6 +1218,20 @@ discuss.</p>
 </div>
 
 
+<h3 id="synchronous">Use synchronous when appropriate</h3>
+
+Synchronous APIs should be considered when minting a new API.
+Providing an API in synchronous form simplifies the usage from the caller's perspective.
+Moreover, it reduces the amount of infrastructural setup overhead needed for the function call.
+
+The general rule of thumb for an API that should be synchronous is the following:
+
+  * The API call is not expected to fail.
```

Comment:
Why does this matter?

Reply:
The synchronous part, or the not-failing part?

Reply:
Not failing. A lot of non-promise returning methods have some kind of failure scenario.

Reply:
Just revisiting this today. Is there a compromise text we could introduce here? @cynthia

Reply:
This was a suggestion from @kenchris - I think we can probably remove that clause and add it back when we have more context.